Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM node:latest as builder
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+ENV PATH /usr/src/app/node_modules/.bin:$PATH
+
+ADD package.json /usr/src/app/package.json
+ADD src /usr/src/app/src
```

Comment:
Przy okazji, czemu ADD a nie COPY? 
https://nickjanetakis.com/blog/docker-tip-2-the-difference-between-copy-and-add-in-a-dockerile

Reply:
Masz racje, przyzwyczajony po prostu jestem do bezmyślnego używania wszędzie ADD...